══════════════════════ Running solver against test cases ══════════════════════
=> fup_001 tests:07 sources:[cases.tio(07)] solvers:[solver.cpp][ ϵ ω ω ϵ ω ω ϵ ]
(ϵ execut_error)[00] GR:014 cases.tio (duas raízes) 
(ω wrong_output)[01] GR:014 cases.tio (duas raízes) 
(ω wrong_output)[02] GR:014 cases.tio (uma raiz   ) 
(ϵ execut_error)[03] GR:014 cases.tio (sem raízes ) 
(ω wrong_output)[04] GR:014 cases.tio (05         ) 
(ω wrong_output)[05] GR:014 cases.tio (06         ) 
(ϵ execut_error)[06] GR:014 cases.tio (07         ) 
╭──────────────────────────────────────────────────────────────────────────────
│             (ϵ execut_error)[00] GR:014 cases.tio (duas raízes)              
├─────────────────────────────────── INPUT ────────────────────────────────────
├────────────────────────────────── EXPECTED ──────────────────────────────────
│ 0.44
│ -5.07
│ 
├────────────────────────────────── RECEIVED ──────────────────────────────────
│ terminate called after throwing an instance of 'std::invalid_argument'
│   what():  Impossivel calcular
│ fail: runtime exception
╰──────────────────────────────────────────────────────────────────────────────

╭──────────────────────────────────────────────────────────────────────────────
│             (ω wrong_output)[01] GR:014 cases.tio (duas raízes)              
├─────────────────────────────────── INPUT ────────────────────────────────────
│ 3.0
│ -7.0
│ 4.0
├────────────────────────────────── EXPECTED ──────────────────────────────────
├ 1.33
├ 1.00
├────────────────────────────────── RECEIVED ──────────────────────────────────
├ 1.33333
├ 1.00000
├────────────────────────────────── MISMATCH ──────────────────────────────────
│ 1.33↵    (expected)
│ 1.33333↵ (received)
│     ↑    (mismatch)
╰──────────────────────────────────────────────────────────────────────────────

╭──────────────────────────────────────────────────────────────────────────────
│             (ω wrong_output)[02] GR:014 cases.tio (uma raiz   )              
├─────────────────────────────────── INPUT ────────────────────────────────────
│ 9.0
│ -12.0
│ 4.0
├────────────────────────────────── EXPECTED ──────────────────────────────────
├ 0.67
├ 
├────────────────────────────────── RECEIVED ──────────────────────────────────
├ 0.66667
├ 0.66667
├────────────────────────────────── MISMATCH ──────────────────────────────────
│ 0.67↵    (expected)
│ 0.66667↵ (received)
│    ↑     (mismatch)
╰──────────────────────────────────────────────────────────────────────────────

╭──────────────────────────────────────────────────────────────────────────────
│             (ϵ execut_error)[03] GR:014 cases.tio (sem raízes )              
├─────────────────────────────────── INPUT ────────────────────────────────────
│ 5.0
│ 3.0
│ 5.0
├────────────────────────────────── EXPECTED ──────────────────────────────────
│ nao ha raiz real
│ 
│ 
├────────────────────────────────── RECEIVED ──────────────────────────────────
│ terminate called after throwing an instance of 'std::invalid_argument'
│   what():  Impossivel calcular
│ fail: runtime exception
╰──────────────────────────────────────────────────────────────────────────────

╭──────────────────────────────────────────────────────────────────────────────
│             (ω wrong_output)[04] GR:014 cases.tio (05         )              
├─────────────────────────────────── INPUT ────────────────────────────────────
│ 1.0
│ -1.0
│ -20.0
├────────────────────────────────── EXPECTED ──────────────────────────────────
├ 5.00
├ -4.00
├────────────────────────────────── RECEIVED ──────────────────────────────────
├ 5.00000
├ -4.00000
├────────────────────────────────── MISMATCH ──────────────────────────────────
│ 5.00↵    (expected)
│ 5.00000↵ (received)
│     ↑    (mismatch)
╰──────────────────────────────────────────────────────────────────────────────

╭──────────────────────────────────────────────────────────────────────────────
│             (ω wrong_output)[05] GR:014 cases.tio (06         )              
├─────────────────────────────────── INPUT ────────────────────────────────────
│ 2.00
│ 0.00
│ 0.00
├────────────────────────────────── EXPECTED ──────────────────────────────────
├ 0.00
├ 
├────────────────────────────────── RECEIVED ──────────────────────────────────
├ 0.00000
├ -0.00000
├────────────────────────────────── MISMATCH ──────────────────────────────────
│ 0.00↵    (expected)
│ 0.00000↵ (received)
│     ↑    (mismatch)
╰──────────────────────────────────────────────────────────────────────────────

╭──────────────────────────────────────────────────────────────────────────────
│             (ϵ execut_error)[06] GR:014 cases.tio (07         )              
├─────────────────────────────────── INPUT ────────────────────────────────────
│ 5.3
│ 3.4
│ 19.2
├────────────────────────────────── EXPECTED ──────────────────────────────────
│ nao ha raiz real
│ 
│ 
├────────────────────────────────── RECEIVED ──────────────────────────────────
│ terminate called after throwing an instance of 'std::invalid_argument'
│   what():  Impossivel calcular
│ fail: runtime exception
╰──────────────────────────────────────────────────────────────────────────────

